modules/core/src/test/scala/org/tessellation/domain/statechannel/StateChannelValidatorSuite.scala

```diff
@@ -229,6 +267,25 @@ object StateChannelValidatorSuite extends MutableIOSuite {
     } yield expect.same(Valid(scOutput), result)
   }
 
+  test("should succeed when no signature is on the state channel allowance list for given address but is validated as historical") { res =>
```

Comment:
Looks like a copy paste of the test above 🤔

Reply:
yes, indeed, but it checks the opposite. the test above should fail, this test should succeed because it's validated as historical snapshot

Reply:
... this one are semantically the same (the name is the same as well)

Reply:
fixed :+1: